src/Sylius/Bundle/CoreBundle/Form/Type/Checkout/PaymentType.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\CoreBundle\Form\Type\Checkout;
+
+use Sylius\Bundle\CoreBundle\Form\EventSubscriber\AddPaymentMethodsFormSubscriber;
+use Sylius\Component\Core\Model\ChannelInterface;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * @author Anna Walasek <anna.walasek@lakion.com>
+ */
+class PaymentType extends AbstractType
```

Comment:
I don't need validationGroups, so I have to ovveride constructor and `configureOptions()` from AbstracResourceType too. For this reason I prefer AbstractType.